_data/accessibility-tests/date-picker.yml

```
@@ -0,0 +1,130 @@
+title: Date picker
+component_status: pass
+test_items:
+# General tests
+  - summary: Touch target size for all clickable elements meets height and width requirements.
+    summary_additional: |
+      When you select any date picker element to inspect and review component code,
+      you can see the total width of the touch target is at least 24 pixels wide and 24 pixels tall.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 2.5.8
+  - summary: Content doesn’t change before submission.
+    summary_additional: |
+      When you enter a date into the date picker,
+      the date stays the same and content doesn't change unless you take another action (e.g., you hit the submit button)
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.2.2
+  - summary: Errors are clearly identified.
+    summary_additional: |
+      If you type in a date range in an incorrect format, the error is identified and described to you in the text.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.1
+  - summary: Text instructions for the date picker are clear.
+    summary_additional: |
+      When you view the date picker, you find clear, simple instructions in text for what type of date format is expected.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 3.3.2
```

Hey @amyleadem  I realized this was a "pass with exception" item.

Should we update to include the explanation and issue?

```suggestion
  - summary: Text instructions for the date picker are clear.
    summary_additional: |
      When you view the date picker, you find clear, simple instructions in text for what type of date format is expected.
    test_status: exception
    test_type: general
    status_details: |
      The date range picker component may not pass all elements of this test.
      The helper text relates to what format is expected of the date, but there are not visible instructions for how to use the calendar button. To meet user needs, you will need to add instructions for each element in the date picker.
    github_issue_number: 6241
    github_issue_repo: uswds
    version_tested: 3.7.1
    wcag_criterion: 3.3.2
```

---

@alex-hull @amycole501, what status do you want to see on this one? Is it a pass or passes with exceptions?

---

Yes, I would mark as passes with exceptions.

---

Updated the test status in [db4b8e7](https://github.com/uswds/uswds-site/pull/3051/commits/db4b8e7d6fa8ab4b72fcb8d56549ec2931877e82)

Please confirm that I added the correct "status details" summary here.